### PR TITLE
Project setup

### LINKED ISSUE #1: [1pt] Create Item Class (Group Task)
* Create Item class in a separate .rb file.
* All Item class properties visible in the diagram should be defined and set up in the constructor method. Exception: properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created.

### LINKED ISSUE #2: [2pt] Implement Item Class Methods (Group Task)
* Add all methods visible in the diagram.
* Implement methods:
can_be_archived?() in the Item class
should return true if published_date is older than 10 years
otherwise, it should return false
move_to_archive() in the Item class
should reuse can_be_archived?() method
should change the archived property to true if the result of the can_be_archived?() method is true
should do nothing if the result of the can_be_archived?() method is false

### LINKED ISSUE #3: [2pt] Create main.rb File (Group Task)
* Create a main.rb file that will serve as your console app entry-point.
* Implement startup actions:
 Present the user with a list of options to perform.
Let users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.

### PR DESCRIPTION
**Pull request by Zia123456**
# Project requirements for initial set-up 🌟
* Create Item class in a separate .rb file.
* All Item class properties visible in the diagram should be defined and set up in the constructor method. Exception: properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created.
* Add all methods visible in the diagram.
* Implement methods:
    * can_be_archived?() in the Item class
        * should return true if published_date is older than 10 years
        * otherwise, it should return false
    * move_to_archive() in the Item class
        * should reuse can_be_archived?() method
        * should change the archived property to true if the result of the can_be_archived?() method is true
        * should do nothing if the result of the can_be_archived?() method is false
* Create a main.rb file that will serve as your console app entry-point.
* Implement startup actions:
    * Present the user with a list of options to perform.
    * Let users choose an option.
    * If needed, ask for parameters for the option.
    * Have a way to quit the app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,118 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Paulina Gonzalez**
+
+- GitHub: [@paulinagonzalezc](https://github.com/paulinagonzalezc)
+- Twitter: [@paulinagzzc](https://twitter.com/paulinagzzc)
+- LinkedIn: [Paulina Gonzalez Cantu](https://linkedin.com/in/paulina-gonzalez-cantu)
+
```
Comment (Zia123456) [inline README.md:82]: - Include all authors
- there are more than one contributors.

**Review by Zia123456 — COMMENTED** [review]
# 🟠 Changes required! 

## Hi @team  👋
image


Your project is incomplete!

There is linters error in your project, please fix it by running ▶ rubocop -A.
 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (paulinagonzalezc) [reply README.md:82]: @storres20 , Can you share your info so we can add it?

**Review by paulinagonzalezc — APPROVED** [review]
✔️ Status: APPROVED 🟢

Hi Team,
 You did a really great job! 💯  Congratulations! 🎉

General requirements:
 ✅ The repo is professional according to the [basic rules] 
✅ Linter error check validated
✅ Gitflow was correctly used
✅ All concepts implemented so far are correct

You may now close this Pull Request and merge the changes!

Optional suggestions 
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Cheers and Happy coding! Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**